.github/workflows/build.yml

```diff
@@ -25,87 +25,36 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres:15
+        env:
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: movies
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          - 5432:5432
     strategy:
       fail-fast: false
       matrix:
         include:
           - scala: 2.12.17
-            scalaShort: "2.12"
-            overall: 0.0
-            changed: 80.0
           - scala: 2.13.12
-            scalaShort: "2.13"
-            overall: 0.0
-            changed: 80.0
     name: Build and test
```

Comment:
Is the matrix needed?

Reply:
Please apply the good boy scout principle and move them to ENV

Reply:
The project does not support cross-version builds now. So this matrix runs all tests on both supported Scala versions.

Reply:
Sorry, my bad, I thought it was scala/spark not scala/scala